clients/cobol-lsp-vscode-extension/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "cobol-language-support",
-	"version": "0.10.0",
+	"version": "0.10.1",
```

Comment:
@sergiuilie wasn't it changed during the last release?

Reply:
it is package-lock, it is updated automatically, no need to commit it